dashboard/src/modules/components/EmptyPageComponent/NoMatchingPage.jsx

```diff
@@ -15,7 +14,7 @@ const NoMatchingPage = () => {
         <img
           className="pbench-logo"
           alt="pbench logo"
-          onClick={() => navigate("/" + HOME)}
+          onClick={() => navigate("/")}
```

Comment:
Will this work the same as `/dashboard/`? That would be great...

Reply:
It's an override while conflict resolving